openapi.yaml

```diff
@@ -181,7 +183,45 @@ paths:
 definitions:
   ProposedEntry:
     type: object
-    $ref: "pkg/types/schemas/base_schema/v0.0.1/base_schema.json"
+    discriminator: kind
+    properties:
+      kind:
+        type: string
+    required:
+      - kind
+
+  rekord:
+    type: object
+    description: Rekord object
+    allOf:
+    - $ref: '#/definitions/ProposedEntry'
```

Comment:
the `allOf` semantics here effectively implements inheritance of `ProposedEntry` for this type